flow/labels.py

```diff
@@ -11,16 +11,17 @@
 
 class label(object):
     """Decorate a :class:`~.FlowProject` class function as a label function.
-
+    
     For example:
-
+    
     .. code-block:: python
-
+    
         class MyProject(FlowProject):
-
+    
             @label()
             def foo(self, job):
                 return True
```

Comment:
This should be replaced by an Examples block with a doctest-style code snippet. Look at the "Examples" here: https://numpydoc.readthedocs.io/en/latest/format.html

Something like:
```
Examples
--------
>>> from flow import FlowProject, label
>>> class MyProject(FlowProject):
...     @label
...     def foo(self, job):
...         return True
```